File: Python Programming/problems on numbericals/problem20.py
'''

20.  Write a program which accept distance in kilometer and convert it into meter. 
		(1 kilometre = 1000 Meter)  

	Author : Milind Nalawade
	
	Date:26/06/2021
	
	Language : Python 3.9.0
	
'''
#======
#Logic
#======

def Kilometer(m):
	return (m*1000);

#=============
#Entry point function
#=============

def main():
	m=int(input("Enter  distance in kilometer : "))
	
	ret=Kilometer(m)
	print("distance in meter is : ",ret,"m")
	
#======
#starter
#======
if __name__=="__main__":
	main()

File: C++ Programming/problems on numberical/problem20.cpp
/*

20.  Write a program which accept distance in kilometre and convert it into meter. 
		(1 kilometre = 1000 Meter)  

	Author : Milind Nalawade
	
	Date:26/06/2021
		
	Language : C++ programming
	
*/	
using namespace std;
#include<iostream>

//Logic
int Kilometer(int m)
{
	return (m*1000);
}

// Entry point function

int main()
{
	int m=0,ret=0;
	cout<<"Enter Kilometer : ";
	cin>>m;
	ret=Kilometer(m);
	printf("Kilometer into meter is : %d m",ret);
	return 0;
}